packages/stable/src/types.ts

```diff
@@ -672,26 +672,23 @@ export type DatapointsDeleteRequest =
   | (ExternalId & DatapointsDeleteRange);
 
 export interface DatapointAggregates extends DatapointsMetadata {
+  isString: false;
+  isStep: boolean;
   datapoints: DatapointAggregate[];
 }
 
 export type Datapoints = StringDatapoints | DoubleDatapoints;
 
 export interface DoubleDatapoints extends DatapointsMetadata {
-  /**
-   * Whether the time series is string valued or not.
-   */
```

Comment:
can you add `isString: boolean;` to `DatapointsMetadata` and add tsdoc there?

Reply:
That will give it a triple comment when hovering over, but it's probably better in total

Reply:
not if it's in `DatapointsMetadata `

Reply:
Yes, I know it sounds wrong but it splits the same comment into three fields with the same name. I think it's better to give each a comment, in such a way that even reading them all concatenated makes sense